web/client/components/data/identify/enhancers/identify.js

```diff
@@ -91,6 +93,9 @@ export const identifyLifecycle = compose(
                 showAllResponses,
                 highlight: pluginCfg?.highlightEnabledFromTheStart || false
             });
+            if (hidePopupIfNoResults) {
+                enableHideEmptyPopupOption(true);
```

Comment:
can't you just use the hidePopupIfNoResults  prop instead of calling enableHideEmptyPopupOption and updating state?

Reply:
@MV88  the point is the cfg 'hidePopupIfNoResults ' is for identify plugin and I used it in map plugin